src/Core/CommandHandler.ts

```diff
@@ -9,9 +9,9 @@ export class CommandHandler {
     private readonly prefix: string;
 
     private constructor(prefix: string) {
+        // Todo : Class should be load dynamically from Commands folder (maybe with namespace ?)
```

Comment:
Please, add a commit for this. Class should be load dynamically  from Commands folder.
@qcarpentier

Reply:
After information on the official Typescript discord, it is **something that is risky to implement in typescript because it does not support dynamic imports**. 

If someone has a solution, make an issue.

Because of this, we will for now just **create a class that will be used to import commands**. When creating a command, **don't forget to import the command into this class**.